packages/core/src/metrics/MetricsService.ts

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+import ServiceConfiguration from "../configuration/ServiceConfiguration";
+import ServiceModule from "../ServiceModule";
+import Metrics from "./Metrics";
+
+class MetricsService implements ServiceModule {
+
+  public readonly type: string = "metrics";
+
+  private readonly url: string;
+
+  constructor(configuration: ServiceConfiguration) {
+    this.url = configuration.url;
+  }
+
+  protected sendAppAndDeviceMetrics(metrics: Metrics): Promise<any> {
+    return axios.post(this.url, metrics);
+  }
+
+  protected getClientId(): string {
+    return "453de743207a0232a339a23e5d64b289";
```

Comment:
Hradcoded? I think that should refer to https://github.com/aerogear/aerogear-android-sdk/blob/master/core/src/main/java/org/aerogear/mobile/core/utils/ClientIdGenerator.java#L31